.gitignore

```diff
@@ -6,3 +6,4 @@ default.etcd
 oauth/client.json
 oauth/user.json
 .idea
+example.eskip
```

Comment:
do you mean this file is generated somewhere during the tests? the json files above are.